app/src/test/java/eu/kanade/tachiyomi/ui/library/CategoryLockManagerTest.kt

```
@@ -8,32 +8,45 @@ import io.kotest.matchers.collections.shouldContain
-        val module = InjektModule {
-            addSingletonFactory { securityPreferences }
+        // Initialize Koin only once with the same mock instance
+        try {
+            startKoin {
+                modules(
+                    module {
+                        single { securityPreferences }
+                    },
+                )
+            }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)